documentation/jetty-documentation/src/main/asciidoc/operations-guide/features.adoc

```diff
@@ -20,6 +20,7 @@ Protocols::
 * xref:og-protocols-http[HTTP/1.1 Support]
 * xref:og-protocols-http2[HTTP/2 Support]
 * xref:og-protocols-websocket[WebSocket Support]
+* xref:og-server-compliance[RFC Compliance Modes]
```

Comment:
@sbordet  I don't think you are fully pushed?  I can't see where og-server-compliance is defined?
Also I think the section should just be "Compliance Modes" rather than RFC

Reply:
@gregw I removed it as now it's prominent in the programming guide, under "Server Compliance Modes".

Reply:
@sbordet I thought the compliance section was a bit burried and hard to find.  So I've given it its own section under the server module.   I think the text of that whole section could do with some spruiking, but I don't mind the structure and general content.